data/schemas/node-type-genspec.schema

```diff
@@ -39,7 +39,7 @@
     },
 
     "data_type": {
-      "enum": [ "boolean", "int", "byte", "string", "float", "rgb", "direction-vector", "error", "location", "timestamp" ]
+      "enum": [ "boolean", "int", "byte", "string", "float", "rgb", "direction-vector", "error", "location", "timestamp", "key-value" ]
```

Comment:
we can't use `key-value` as both strings if we plan to support other types of values in the future. Better call it `key-string-value` or `key-value-string`.